src/main/java/at/nightfirec/wildernessmultilines/WildernessMultiLinesOverlay.java

```diff
@@ -60,6 +61,12 @@ private WildernessMultiLinesOverlay(WildernessMultiLinesPlugin plugin, Wildernes
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		boolean inWilderness = client.getVarbitValue(Varbits.IN_WILDERNESS) == 1;
+		if (config.onlyShowInWilderness() && !inWilderness)
```

Comment:
Var check before config check to reduce config hits
```suggestion
		final boolean inWilderness = client.getVarbitValue(Varbits.IN_WILDERNESS) == 1;
		if (!inWilderness && config.onlyShowInWilderness())
```